docs/docsite/rst/dev_guide/ansible_index.rst

```diff
@@ -59,7 +59,7 @@ Find the task that best describes what you want to do:
 
   * I want to :ref:`understand how to contribute to Ansible <ansible_community_guide>`.
   * I want to :ref:`contribute my module or plugin <developing_modules_checklist>`.
-  * I want to :ref:`understand the license agreement <contributor_license_agreement>` for contributions to Ansible.
+  * I want to :ref:`understand the license agreement <developer_certificate_of_origin>` for contributions to Ansible.
```

Comment:
Shouldn't here `license` be changed to `DCO` as well (as in docs/docsite/rst/dev_guide/developing_modules_checklist.rst)? Same for the next ones.

Or add `(DCO)` in brackets, so that folks know that this isn't some CLA?

Reply:
Good catch, fixed